providers/ms365/resources/ms365.lr

```diff
@@ -147,6 +149,52 @@ private microsoft.identityAndAccess.roleEligibilityScheduleInstance @defaults("m
   roleEligibilityScheduleId string
 }
 
+// A filterable list of access review definitions
+microsoft.identityAndAccess.accessReviews {
+  []microsoft.identityAndAccess.accessReviewDefinition
+  init(filter? string)
+  // filter access review definitions. e.g., "contains(scope/microsoft.graph.accessReviewQueryScope/query, './members')"
+  filter string
+}
+
+// Represents the definition of an access review
```

Comment:
Try to avoid starting the description of the resources themselves with "Represents". It's a bit redundant and makes it harder to read into what the resource does at a glance